Wikipedia/Code/WMFAppViewController.m

```diff
@@ -783,7 +785,7 @@ - (void)migrateIfNecessary {
                 [self checkRemoteAppConfigIfNecessary];
                 [self setupControllers];
                 if (!self.isWaitingToResumeApp) {
-                    [self resumeApp:NULL];
+                    [self resumeApp:[self presentLanguageVariantAlerts]];
```

Comment:
Sorry, the best area to trigger this is more complicated than I realized. I'll do some testing to see if any changes are needed, but for initial code review I would say maybe just at a "WithCompletion" name change to this `resumeApp` method, since it's a bit confusing on what's going on here without it, if we decide to keep this placement.

It looks like `finishResumingApp` can trigger various panels to display like `showLoggedOutPanelIfNeeded`. This method is called within `resumeApp`, before the completion block is called, so my concern is this variant modal will not play nicely with those if they end up being triggered at the same time. I'll confirm that it's a problem first though, just something that jumped out at me.